packages/components/src/components/textarea/textarea.tsx

```diff
@@ -186,6 +194,11 @@ export class Textarea {
     if (this.hasCounter) {
       this.setCounterAriaTextDebounced();
     }
+    this.internals.setValidity(
```

Comment:
Does this work if the textarea is disabled or do we need the same condition like in the checkbox?

Reply:
> TypeError: Failed to execute 'setValidity' on 'ElementInternals': The second argument should not be empty if one or more flags in the first argument are true.

This error is only reproducible for the checkbox. If you set the textarea to `disabled` & `required` the `ValidityState` will contain `{valueMissing: false, ...}` while it will be `{valueMissing: true, ...}` for the checkbox. Therefore we do not need the condition like we have for the checkbox